File: repetitivas/ej4.py
# -*- coding: utf-8 -*-

#Escribir un programa que imprima 
#todos los números pares entre dos números que se le pidan al usuario.

#almacenar inicio del rango de enteros
#almacenar fin del rango de enteros
#calcular módulo 2 del elemento del rango
#si el resultado es 0, por lo tanto es par, se muestra

print("..:: Mostrar números pares de un rango ::..")
inicio = int(input("Introduce el valor de inicio del rango> "))
fin = int(input("Introduce el valor final del rango> "))
iterador = inicio
for iterador in range(inicio,fin):
	if(iterador%2==0):#si el resultado del módulo 2 es 0, es par
		print(iterador)

File: Java/Repetitivas/ej4.java
import java.util.Scanner; //Importar la clase Scanner

/**
 * Escribir un programa que imprima todos los números pares 
 * entre dos números que se le pidan al usuario.
 * @author Alberto Miguel Martinez Jimenez
 */
/**
 * Algoritmo
 * int numeroInicio;
 * int numeroFin;
 * if numero%2=0, mostrar número
 */

public class ej4 {

	public static void main(String[] args) {
		int numeroInicio;
		int numeroFin;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("-Mostrar números pares en un rango-");
		System.out.println("número de comienzo de rango> ");
		numeroInicio = scanner.nextInt();
		System.out.println("número de fin de rango> ");
		numeroFin = scanner.nextInt();
		
		for(int i=numeroInicio;i<=numeroFin;i++) {
			if(i%2==0) {
				System.out.println(i);
			}
		}
	}

}
